src/encoded/tests/test_audit_biosample.py

```diff
@@ -130,6 +20,7 @@ def test_audit_biosample_modifications_whole_organism(
     assert any(error['category'] == 'mismatched genetic modifications' for error in errors_list)
 
 
+@pytest.mark.skip(reason='will be addressed in a PYTEST-REFACTOR')
 def test_audit_biosample_modifications_whole_organism_duplicated(
```

Comment:
I think this test assert is wrong, it should have been any( blah blah == blah blah) the "!=" is supposed to be "=="

Reply:
Thanks. Done.